include/libsemigroups/knuth-bendix.tpp

```diff
@@ -776,6 +778,10 @@ namespace libsemigroups {
   template <typename Rewriter, typename ReductionOrder>
   typename detail::internal_char_type
   KnuthBendix<Rewriter, ReductionOrder>::uint_to_internal_char(size_t a) {
+    // TODO What is this check for?
```

Comment:
The check is just to ensure that the input value doesn't overflow the internal char type, seems legit to me